### PR TITLE
environment variables moved to separate module

### PR DESCRIPTION
**Pull request by decobots**
issue #7

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -5,12 +5,7 @@
 from xml.etree import ElementTree as etree
 from urllib.parse import urlencode
 import json
-import os
-
-CONSUMER_KEY = os.getenv("CONSUMER_KEY")
-CONSUMER_SECRET = os.getenv("CONSUMER_SECRET")
-ACCESS_KEY = os.getenv("ACCESS_KEY")
-ACCESS_SECRET = os.getenv("ACCESS_SECRET")
+from environment_variables import *
```
Comment (livercat) [inline main.py]: Using *-imports is heavily discouraged: you can't at a glance understand from the script which names are imported, and if you import * from 2+ modules, it becomes a nightmare. Just list all imports by name.

File: environment_variables.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import os
+
+CONSUMER_KEY = os.getenv("CONSUMER_KEY")  # twitter
```
Comment (livercat) [inline environment_variables.py]: CONSUMER_KEY -> TWITTER_CONSUMER_KEY, etc. and remove comments

**Comment by livercat** [comment]
Bonus tip: use Ctrl+Alt+O in addition to Ctrl+Alt+L in Pycharm to optimize imports (sorts and removes unused ones).

File: main.py (hunk 1 above)
Comment (decobots) [reply main.py]: I thought it's so cool not to write this long line of names,  but I see your point.

File: environment_variables.py (hunk 2 above)
Comment (decobots) [reply environment_variables.py]: Fixed, thank you!

**Comment by decobots** [comment]
Bonus tip: use Ctrl+Alt+O in addition to Ctrl+Alt+L in Pycharm to optimize imports (sorts and removes unused ones).
1) I don't understand how to replay on comment at gitHub(
2) I don't want to do it in this branch, because I changed imports in another.  I created issue #12 to improve it later.

**Pull request merged**